### ISSUE TITLE
filepath is different from what I picked

### ISSUE THREAD

**Issue by CristianKerr** [alexkdeveloper/desktop-files-creator#38]
Hi,
I select path to sh or png file in exec/icon, but the path is different.
E.g.
home/me/Apps/MyApp/run.sh
but the path that is fille is e.g.
/run/user/1000/doc/41389e01/run.sh
I guess this has to do with flatpak, but I do not know how to fix it.

**Comment by alexkdeveloper** [comment]
Hi!
This is a consequence of the fact that the application gets access to files through portals. The application does not have access to the user's files.

But you can give this permission using the [Flatseal](https://flathub.org/apps/com.github.tchx84.Flatseal) app. Then maybe the file paths will change.

Although I don't quite understand why to do this. Everything should work anyway.

**Comment by CristianKerr** [comment]
Hi, thanks for fast answer.
I checked and it did copy the sh file into the portal location which made the program not work because it didn't have the files it needed (it was RustRover from Jetbrains just fyi).
When I set the permission, the path is now correct and the shortcut works. 
Thank you :)

Edit: anyway this is more flatpak issue than the program issue, so we can close this, thanks again!
